Resources/Markdown/docs/builds.md

```diff
@@ -86,3 +87,11 @@ In cases like this, Xcode cannot resolve package dependencies because of how it
 ```
 
 You can [find more details in this issue](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/1532).
+
+<h3 id="data-race-safety">What is data race safety and how is it tested?</h3>
+
+The Swift 6.0 compiler can test whether code is safe from data races at compile time when strict concurrency checks are enabled. The data race safety information we publish on package pages comes from metadata output by the compiler during our build process.
```

Comment:
```suggestion
The Swift 6.0 compiler can check whether code is safe from data races at compile time when strict concurrency checks are enabled. The data race safety information we publish on package pages comes from metadata output by the compiler during our build process.
```

probably best to stick to "check" in terms of compiler and leave "test" wording to tests.